google-cloud-tools-plugin/src/com/google/gct/idea/debugger/CloudDebugGlobalPollerTimerTask.java

```diff
@@ -0,0 +1,22 @@
+package com.google.gct.idea.debugger;
+
+import java.util.TimerTask;
+
+/**
+ * Polls changes from the Cloud Debug backend executed by a timer in CloudDebugGlobalPoller
```

Comment:
Polls for changes from the Cloud Debugger.  You don't want to refer to your client classes in your Javadoc.